src/rules/strictTypePredicatesRule.ts

```diff
@@ -52,6 +54,10 @@ export class Rule extends Lint.Rules.TypedRule {
     }
 
     public applyWithProgram(sourceFile: ts.SourceFile, program: ts.Program): Lint.RuleFailure[] {
+        if (!Lint.isStrictNullChecksEnabled(program.getCompilerOptions())) {
+            showWarningOnce("strict-type-predicates does not work without strictNullChecks");
```

Comment:
I think it's fine. Would probably be more clear if it said `tsc option --strictNullChecks`, but if you search for `strictNullChecks`, the first hit is correct